### PR TITLE
Jordan nash fix crud md dev

### PR DESCRIPTION
**Pull request by LayersOfAbstraction**
Hi mate, sorry but I tried to merge this with the original branch I made for #27256 but that did not seem to merge properly so I had to put my changes into this new branch which we can focus on from now on. Keep in mind I am new to merging. I am new to using the DocFX tool on Windows so I have not been able to figure out how display .NET code snippets just yet due to possible configuration errors. 

Changes I have made to the project include:

1. change all asp-validation-summary from:
```<div asp-validation-summary="ModelOnly" class="text-danger"></div>```
to this:
```<div asp-validation-summary="All" class="text-danger"></div>``` as you advised. Did this both for Create.cshtml and Edit.cshtml in the Student folder. 

2. Added snippet showing nullable navigational property in Student.cs.
3. In Crud.md I added highlighted line for nullable navigational property in Student.cs and highlighted lines for changing ```asp-validation-summary helper tag``` to ```All``` for ```Edit.cshtml``` and ```Create.cshtml``` in the Views/Students folder. 
4. Also mentioned how user could comment out global nullable property globally.

### CODE REVIEW

File: .github/workflows/dependencies/run-dotnet-whatsnew.sh [hunk 1]
```diff
@@ -20,4 +20,4 @@ done
 dotnet whatsnew \
     --owner $OWNER \
     --repo $REPO \
-    --savedir $SAVEDIR
+    --savedir $SAVEDIR
```
Comment (guardrex) [inline .github/workflows/dependencies/run-dotnet-whatsnew.sh:23]: @LayersOfAbstraction ... Add the return back to this line ...

```suggestion
    --savedir $SAVEDIR

```

**Review by LayersOfAbstraction — COMMENTED** [review]
I'm supposed to review this change? Oh ok thanks, I'm just reading about all this now [in GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews). I didn't know I made any changes here. Must have hit the enter button without even knowing before in this file. Let me know if there is anything further I need to do before this gets accepted, thanks!

**Comment by guardrex** [comment]
It was the other way around ... the existing return was deleted ... i.e., probably a stray backspace 😈.

It was either that **_--OR--_** ... *That was the 👽 signal to begin the attack on Earth!* Thank goodness we stopped that. 😄

**Comment by guardrex** [comment]
That was already resolved by my suggestion and Rick accepting it. I think the file is broken again. Look at the diff ... there shouldn't be a red negative sign there. I wonder if your line endings are freaking out GH. I'll edit the file manually from here to see if I can fix it. *Stand-by ...*

**Comment by LayersOfAbstraction** [comment]
@Rick-Anderson because you have not yet published the pull request, I am concerned I have inadvertently fried your brain with what I suppose is a **big** pull request too detailed for one reviewer. 

Did you need me to break things down a bit? You know like: 

1. Have all the changes I made to the documentation file `crud.md` stay in this pull request only and...
 
2. Make a new branch containing an update to only the ContosoUniversity sample app files i.e the model and view to reflect what they should look like in the `curd.md` file. 

3. Revert all the ContonosoUnivesity model and view files back to what they were pre-commit **in this branch only** so it's only focused on the documentation and not the actual sample app?
   
Else do you just need more time, please?

**Comment by Rick-Anderson** [comment]
@LayersOfAbstraction  can you apply your changes to #27440 to this PR? I think we'll have time to review this soon.
cc @tdykstra

**Comment by LayersOfAbstraction** [comment]
> #27440

You mean you want me to merge all the changes for Crud.md from this PR into #27440 so you can close this PR? Sure I'll do this after my shift tonight and cc tdykstra in the PR, thanks.

**Comment by Rick-Anderson** [comment]
> > #27440
> 
> You mean you want me to merge all the changes for Crud.md from this PR into #27440 so you can close this PR? Sure I'll do this after my shift tonight and cc tdykstra in the PR, thanks.

If it's not too much work. Maybe @tdykstra  is OK with 2 PR's.

**Comment by LayersOfAbstraction** [comment]
@Rick-Anderson yeah it should be fine. I will close this.

**Pull request closed without merging**